mobile/src/main/java/com/genymobile/roomconnect/roomengine/JoinRoomListener.java

```diff
@@ -0,0 +1,9 @@
+package com.genymobile.roomconnect.roomengine;
+
+
+public interface JoinRoomListener {
+
+    void onRoomJoinSuccessfull();
+
+    void onRoomJoinError();
```

Comment:
the interface name is prefixed with `JoinRoom` but the method are prefixed with `RoomJoin`. Choose :-)